### PR TITLE
Add PyDESeq2

### PR DESCRIPTION
**Pull request by mikecuoco**
Created using `grayskull pypi pydeseq2`. Package is hosted at [github.com/owkin/PyDESeq2](https://github.com/owkin/PyDESeq2), developed by @BorisMuzellec with help from others.

### CODE REVIEW

**Comment by mikecuoco** [comment]
@BiocondaBot please add label. @BorisMuzellec do you think it is OK to add PyDESeq2 to bioconda at this stage?

File: recipes/pydeseq2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - anndata ==0.8.0
```
Comment (bgruening) [inline recipes/pydeseq2/meta.yaml:23]: Is this strict pin needed?

File: recipes/pydeseq2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+about:
+  home: https://github.com/owkin/PyDESeq2
+  summary: A python implementation of DESeq2.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - mikecuoco
```
Comment (bgruening) [inline recipes/pydeseq2/meta.yaml:48]: please add the DOI

File: recipes/pydeseq2/meta.yaml (hunk 1 above)
Comment (mikecuoco) [reply recipes/pydeseq2/meta.yaml:23]: Yes, until next version is released

File: recipes/pydeseq2/meta.yaml (hunk 2 above)
Comment (mikecuoco) [reply recipes/pydeseq2/meta.yaml]: Done!

**Comment by mikecuoco** [comment]
@bgruening this is ready for another review

**Comment by bgruening** [comment]
What is with your question to @BorisMuzellec ?

**Comment by mikecuoco** [comment]
Boris is the original author of this package. I just wanted to notify him that I was adding this recipe.

**Comment by BorisMuzellec** [comment]
Hi @mikecuoco, sure, sounds great! 
We're still actively developing pydeseq2 though (chasing bugs and adding some features from time to time). Do I have anything to do when a new version is added on PyPI? I'm not very familiar with bioconda

**Review by mjsteinbaugh — APPROVED** [review]
Sounds OK to merge to me

**Pull request merged**